src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/Platform.java

```diff
@@ -11,33 +11,42 @@
 public interface Platform {
     /**
     * if the Platform has his own payment service the implementation can return not none.
-    * @return The value to indicate if it supports self paying or not.
+    * @return the value to indicate if it supports self paying or not.
     */
     Optional<Payment> getPayment();
 
     /**
      * if the Platform has his own worker identification the interface can be returned here
-     * @return The value to indicate if it supports worker identification or not.
+     * @return the value to indicate if it supports worker identification or not.
      */
     Optional<WorkerIdentification> getWorker();
 
     /**
      * Get the name of this platform
-     * @return A unique String
+     * @return a String
      */
     String getName();
 
     /**
+     * Returns the type of the platform.
+     * <p>
+     * The String must match {@code [a-z0-9_]+} and must be a constant since it is the only way other services can
+     * communicate with the platform.
+     * @return a String
+     */
+    String getType();
```

Comment:
is the type now a "type" like mturk or something like a "id" like mturksandbox ?